test/Pool.test.ts

```diff
@@ -215,30 +211,96 @@ describe("Pool Events", () => {
           chainId: 10,
           logIndex: 1,
           srcAddress: "0x3333333333333333333333333333333333333333",
-        },
+        }
+      };
+      beforeEach( async () => {
+        mockLiquidityPoolData.token0_id = TokenIdByChain("0x9999999999999999999999999999999999999990", 10);
+        mockLiquidityPoolData.token1_id = TokenIdByChain("0x9999999999999999999999999999999999999999", 10);
+
+        const updatedDB1 =
+          mockDb.entities.LiquidityPoolNew.set(mockLiquidityPoolData as LiquidityPoolNew);
+
+        const mockEvent = Pool.Swap.createMockEvent(eventData);
+
+        postEventDB = await Pool.Swap.processEvent({
+          event: mockEvent,
+          mockDb: updatedDB1,
+        });
       });
 
-      const result = await Pool.Swap.processEvent({ event: mockEvent, mockDb: updatedDB3 });
+      /**
+       * Data should remain unchanged for non-whitelisted tokens. We want to capture pool data but not prices on tokens.
+       */
+      it("should not update the liquidity pool with swap volume", async () => {
+        const updatedPool = postEventDB.entities.LiquidityPoolNew.get(
+          toChecksumAddress(eventData.mockEventData.srcAddress)
+        );
+        expect(updatedPool).to.not.be.undefined;
+        expect(updatedPool?.totalVolume0).to.equal(mockLiquidityPoolData.totalVolume0);
+        expect(updatedPool?.totalVolume1).to.equal(mockLiquidityPoolData.totalVolume1);
+        expect(updatedPool?.totalVolumeUSD).to.equal(mockLiquidityPoolData.totalVolumeUSD);
+        expect(updatedPool?.numberOfSwaps).to.equal(mockLiquidityPoolData.numberOfSwaps);
+        expect(updatedPool?.lastUpdatedTimestamp).to.equal(mockLiquidityPoolData.lastUpdatedTimestamp);
```

Comment:
The datapoint checks here are equality checks so the liquidity pool is not updated from the initial data.